html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Projects</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="/projects.html">Projects</a></li>
```

Comment:
Hmmm, I'm not sure that this link would work 🤔 . You should test this, but you likely need to remove the `/` at the beginning of the path

Reply:
Oops! Thank you! Just re-submitted the pull request.